src/reducers/ticketReducers.js

```diff
@@ -51,9 +51,13 @@ const removeTicketFromState = (state, ticketId) => {
   return newState
 }
 
-const removeAllTicketsFromState = (state) => {
-  let newState = { ...state }
-  Object.keys(state).map(ticketId => { newState[ticketId] = null })
+const removePreviousTicketsFromState = (state, currentTicketId) => {
+  let newState = {}
+  Object.keys(state)
```

Comment:
_nitpick_ I'd prefer using `map` from lodash or underscore to create this object of keys with null values. Something like:

``` javascript
let newState = _.chain(state).map((_v, key) => [key, null]).fromPairs().value()
```

Reply:
I prefer to avoid bringing in lodash or underscore for things that can be done with the ES6 builtins. That said, I wasn't happy with the way this function was working. I changed it to:

```JavaScript
const removePreviousTicketsFromState = (state, currentTicketId) =>
  Object.assign(
    {},
    ...Object.keys(state)
      .map(ticketId => ({[ticketId]: null})),
    {[currentTicketId]: state[currentTicketId]}
  )
```